backend/src/case_file/case_file.service.ts

```diff
@@ -904,5 +909,193 @@ export class CaseFileService {
       throw new GraphQLError("Exception occurred. See server log for details", {});
     }
   };
+
+  // find all equipment records, and their respective actions, for a given case
+  // Since we want to list the equipment related to a case, rather than the actions for a case, which may contain equipment, let's
+  // transform the actions with equipment to equipment with actions.
+  async _findEquipmentDetails(caseIdentifier: string): Promise<Equipment[]> {
```

Comment:
this function isn't a bad idea, but you might want to look at the `getActions` function to see if this does what you're looking for, if it doesn't then you'll want to move your `_findEquipmentDetails` function into the `findOne` function this way it can't be reused by anyone